### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+* Add support for LBaaS (GH-2)
+* Add support for PaaS (GH-6)
+* Add support for ISO Image Handling (GH-8)
+* Add support for Object Storage (GH-11)
+* Add support for Snapshots (GH-12) and Snapshot Scheduler (GH-13)
+* Add support for Firewall Handling (GH-14)
+
+IMPROVEMENTS:
+
+* Avoid use of fmt.errorf
```
Comment (itakouna) [inline CHANGELOG.md]: We still use `fmt.errorf` in some lines.

Comment (fkr) [reply CHANGELOG.md]: hehe, then it war re-introduced. because at some point I deliberately removed it ;)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+* Unit Tests for all functionality
+* Logging support
```
Comment (itakouna) [inline CHANGELOG.md]: Logging was exist before, but we added the capability to switch between Logging level between Debug and Info.

Comment (fkr) [reply CHANGELOG.md]: yes, that's what I call "logging support" - everything else is just printing stuff.

File: README.md [hunk 3]
```diff
@@ -46,76 +47,169 @@ servers := client.GetServerList()
...
+* Server (server.go)
+* Storage (storage.go)
+* Storage snapshot (snapshot.go)
+* Storage snapshot schedule (snapshotschedule.go)
+* SSH-key (sshkey.go)
+* Template (template.go)
 
 ## Implemented API Endpoints
 
 Not all endpoints have been implemented in this client, but new ones will be added in the future. Here is the current list of implemented endpoints and their respective function written like endpoint (function):
```
Comment (itakouna) [inline README.md]: is it still valid? we did implement them!

Comment (fkr) [reply README.md]: Marketplace is still missing.

**Pull request merged**